kiarash/tsconfig.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
Please do not include these autogenerated files as-is. Edit them and only commit what is really needed.